src/Compilers/CSharp/Test/Emit/PDB/PDBUsingTests.cs

```diff
@@ -525,10 +525,10 @@ class A { void M() {  } }
                 options: TestOptions.DebugDll,
                 references: new[]
                 {
-                    // first aliased reference
-                    compilation1.ToMetadataReference(ImmutableArray.Create("X")),
-                    // second unaliased reference 
-                    compilation1.ToMetadataReference(),
+                // first aliased reference
```

Comment:
Array Initializers are special cases. We are never supposed to disturb a existing Array Initializer.I tried a simpler repro and things seem to be fine.

`Formatter.Format` picks up the default option from the workspace. There is a possibility of some other operation changing the option in the workspace prior to running `Formatter.Format` or a new rule being introduced to the set of formatting rules passed down to the formatting engine or a bug in Roslyn. Without knowing what options, and rules were run, it is tough to find the issue. This is a bug and it would be best if we track it by opening one.